using-valtimo/command-handling/command-handling.md

```diff
@@ -0,0 +1,69 @@
+# Command handling
```

Comment:
This PR adds a lot of new pages, but none of them can be accessed. Can you add them to `SUMMARY.md`? That's what builds up the sidebar.

Reply:
Additionally, can you move the documentation to Extending Valtimo rather than Using Valtimo? As this requires custom code.

Any non-custom code sections can stay in Using Valtimo.

Reply:
will do